docs/root/configuration/access_log.rst

```diff
@@ -228,6 +228,13 @@ The following command operators are supported:
   Local address of the upstream connection. If the address is an IP address it includes both
   address and port.
 
+%UPSTREAM_TRANSPORT_FAILURE_REASON%
```

Comment:
Can we have parity here with the gRPC access log? I would like to make sure we have consistency with all of these types of changes moving forward.

Reply:
Done